defaults/config.js

```diff
@@ -139,6 +139,29 @@ module.exports = {
 	// This value is set to `2048` kilobytes by default.
 	prefetchMaxImageSize: 2048,
 
+	// ### `upload`
+	//
+	// Allow uploading files to the server hosting TheLounge.
+	// If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/uploads` folder.
+	//
+	// Images will not expire or be removed by TheLounge, so be wary of memory and your hardware.
+	//
+	// This value is set to `false` by default.
+	upload: false,
+
+	// ### `uploadMaxFileSize`
+	//
+	// Maximum file size for uploaded files
+	//
+	// If upload is enabled, The Lounge will only allow uploading files with a size < the value below
```

Comment:
This seems redundant with the line above. Instead, I would phrase it to explain what happens when a file above this limit is sent. Something like:
`When file upload is enabled, users sending files above this limit will be prompted an error message in their browser.` (I have no clue if this is remotely accurate as I have not reviewed the rest of the PR yet, it's just to illustrate).